2019-11-21 11:55:48,274 -  common.basepage -INFO - 等待元素 ('xpath', '//input[@name="phone"]') 可见。 - [basepage.py:24 ]
2019-11-21 11:55:48,329 -  common.basepage -INFO - 开始等待时间点：2019-11-21 11:55:48.275959，结束等待时间点：2019-11-21 11:55:48.329959，等待时长为：0:00:00.054000 - [basepage.py:32 ]
2019-11-21 11:55:48,329 -  common.basepage -INFO - 查找 输入手机号 中的元素 ('xpath', '//input[@name="phone"]')  - [basepage.py:64 ]
2019-11-21 11:55:48,337 -  common.basepage -INFO -  给元素 ('xpath', '//input[@name="phone"]') 输入文本内容:('17625188013',) - [basepage.py:122 ]
2019-11-21 11:55:48,451 -  common.basepage -INFO - 等待元素 ('xpath', '//input[@name="phoneCode"]') 可见。 - [basepage.py:24 ]
2019-11-21 11:55:48,478 -  common.basepage -INFO - 开始等待时间点：2019-11-21 11:55:48.451959，结束等待时间点：2019-11-21 11:55:48.478959，等待时长为：0:00:00.027000 - [basepage.py:32 ]
2019-11-21 11:55:48,478 -  common.basepage -INFO - 查找 输入验证码 中的元素 ('xpath', '//input[@name="phoneCode"]')  - [basepage.py:64 ]
2019-11-21 11:55:48,487 -  common.basepage -INFO -  给元素 ('xpath', '//input[@name="phoneCode"]') 输入文本内容:('sD2q',) - [basepage.py:122 ]
2019-11-21 11:55:48,541 -  common.basepage -INFO - 等待元素 ('xpath', '//a[@class="login"]') 可见。 - [basepage.py:24 ]
2019-11-21 11:55:48,571 -  common.basepage -INFO - 开始等待时间点：2019-11-21 11:55:48.541959，结束等待时间点：2019-11-21 11:55:48.571959，等待时长为：0:00:00.030000 - [basepage.py:32 ]
2019-11-21 11:55:48,571 -  common.basepage -INFO - 查找 登录 中的元素 ('xpath', '//a[@class="login"]')  - [basepage.py:64 ]
2019-11-21 11:55:48,578 -  common.basepage -INFO -  点击元素 ('xpath', '//a[@class="login"]') - [basepage.py:89 ]
2019-11-21 12:00:53,576 -  common.basepage -INFO - 等待元素 ('xpath', '//input[@class="s-input"]') 可见。 - [basepage.py:24 ]
2019-11-21 12:05:53,725 -  common.basepage -ERROR - 等待元素可见失败： - [basepage.py:35 ]
Traceback (most recent call last):
  File "D:\B2B\common\basepage.py", line 28, in wait_eleVisible
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "E:\Python3\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "E:\Python3\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 128, in __call__
    return _element_if_visible(_find_element(driver, self.locator))
  File "E:\Python3\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 415, in _find_element
    raise e
  File "E:\Python3\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 411, in _find_element
    return driver.find_element(*by)
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout
  (Session info: chrome=73.0.3683.103)
  (Driver info: chromedriver=2.36.540470 (e522d04694c7ebea4ba8821272dbef4f9b818c91),platform=Windows NT 6.1.7601 SP1 x86_64)

2019-11-21 12:10:53,800 -  common.basepage -ERROR - 网页截屏失败！ - [basepage.py:186 ]
Traceback (most recent call last):
  File "D:\B2B\common\basepage.py", line 28, in wait_eleVisible
    WebDriverWait(self.driver,timeout,frequency).until(EC.visibility_of_element_located(loc))
  File "E:\Python3\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "E:\Python3\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 128, in __call__
    return _element_if_visible(_find_element(driver, self.locator))
  File "E:\Python3\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 415, in _find_element
    raise e
  File "E:\Python3\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 411, in _find_element
    return driver.find_element(*by)
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout
  (Session info: chrome=73.0.3683.103)
  (Driver info: chromedriver=2.36.540470 (e522d04694c7ebea4ba8821272dbef4f9b818c91),platform=Windows NT 6.1.7601 SP1 x86_64)


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\B2B\common\basepage.py", line 183, in save_web_screenshot
    self.driver.save_screenshot(screenshot_dir +"/" + filepath)
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1055, in save_screenshot
    return self.get_screenshot_as_file(filename)
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "E:\Python3\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout
  (Session info: chrome=73.0.3683.103)
  (Driver info: chromedriver=2.36.540470 (e522d04694c7ebea4ba8821272dbef4f9b818c91),platform=Windows NT 6.1.7601 SP1 x86_64)

